Documentation/building/unix-instructions.md

```diff
@@ -59,3 +59,19 @@ If you see errors along the lines of `SendFailure (Error writing headers)` you m
 ```sh
 mozroots --import --sync
 ```
+
+## Steps to Build on Ubuntu
+
+*Note: verified on Ubuntu 14.04 LTS*
+
+1. Install git `sudo apt-get install git`
+2. Clone the corefx repo `git clone https://github.com/dotnet/corefx.git`
+3. Install curl-dev `sudo apt-get install libcurl4-openssl-dev`
```

Comment:
If these commands are supposed to be copy-pasted, wouldn't it be easier to combine all the install commands into one? I.e. `sudo apt-get install git libcurl4-openssl-dev libicu52 llvm lldb-3.5 cmake clang-3.6 libunwind8-dev`.